go/def.bzl

```diff
@@ -937,6 +937,7 @@ def cgo_library(name, srcs,
       copts = copts + [
           "-I", pkg_dir,
           "-I", "$(GENDIR)/" + pkg_dir + "/" + cgogen.outdir,
+          "-pthread",
```

Comment:
This appears to work for both linux and OSX, and rules_go doesn't have Windows support anyway.  Maybe add a TODO to fix if/when Windows support is added?